utils/build-script-impl

```diff
@@ -2124,9 +2348,29 @@ for deployment_target in "${STDLIB_DEPLOYMENT_TARGETS[@]}"; do
         echo "--- Finished tests for ${product} ---"
     done
 done
+# END OF TEST PHASE
 
-for deployment_target in "${HOST_TARGET}" "${CROSS_COMPILE_TOOLS_DEPLOYMENT_TARGETS[@]}"; do
-    set_deployment_target_based_options
+
+LIPO_SRC_DIRS=()
+
+for host in "${ALL_HOSTS[@]}"; do
+
+    # Calculate the directory to install products in to.
+    # If INSTALL_DESTDIR not given, we can't install anything.
+
+    if [[ -z "${INSTALL_DESTDIR}" ]] ; then
+        echo "--install-destdir was not given. Not installing any products."
```

Comment:
Sorry, this is an unfortunate behavior change.  Previously if the user requested installing something (via `--install-*` flags) but did not provide a directory, we produced an error.  Now we would just skip the installation, which is probably not what they wanted.  I'm afraid this would hide errors or make it harder that necessary to debug issues.